Tests/scripts/validate_files_structure.py

```diff
@@ -682,6 +703,203 @@ def str2bool(v):
         raise argparse.ArgumentTypeError('Boolean value expected.')
 
 
+def is_text_file(file_path):
+    file_extension = os.path.splitext(file_path)[1]
+    text_file_types = set(['.yml', '.py', '.json', '.md', '.txt', '.sh', '.ini', '.eml', '', '.csv'])
```

Comment:
why are you converting a list to a set and not just creating a set in the first place?
how did you choose these file types?

Reply:
No particular reason, used same conventions that already exist in the code.

Reply:
ok so let's remove the list there and try to understand where these files types came from

Reply:
I ment no particular reason for the first question, for the second question I forgot to anwer. I chose these file types to filter only text files that we are normally using in the repo. Since I don't want to try and parse files like pictures etc.